doc/client_lua_api.txt

```diff
@@ -30,6 +30,16 @@ Startup
 Mods are loaded during client startup from the mod load paths by running
 the `init.lua` scripts in a shared environment.
 
+In order to load client-side mods in a world, the following conditions need to be satisfied:
+
+ 1) $path_user/minetest.conf contains the setting `enable_client_modding = true`
+
+ 2) The client-side mod located in $path_user/clientmods/<modname> is added to
+    $path_user/clientmods/mods.conf as `load_mod_<modname> = true` *.
+
+* Note: Depending on the remote server's settings, client-side mods might not
```

Comment:
Is the `*` at the beginning of the line required?

Reply:
i don't know for 2) if server can reject a mod based on name if not then yeah * is not relevant.

Reply:
Ah, I just realised that you added the asterisk as a footnote indicator. But I don't think this is required, and could potentially add confusion (like it did for me until a couple of seconds ago). Also, our documentation files follow markdown format (in spite of having `.txt` extensions), and a `*` at the beginning of the line would render as a bullet point. It's best to avoid stray asterisks in general - you'd never know how they'd be interpreted and rendered. ;)